mysql-schema/mysql-schema.md

```diff
@@ -118,6 +126,7 @@ Currently, the `help_topic` is NULL.
 - `opt_rule_blacklist`: the blocklist for logical optimization rules
```

Comment:
https://github.com/pingcap/docs/blob/a468479f3af0f6aa4686ef094e54fa75a2879984/mysql-schema/mysql-schema.md?plain=1#L118-L122

@Defined2014 Could you please confirm whether `advisory_locks` is applicable to/supported in TiDB Cloud? Thank you.

Reply:
Not sure about it. Maybe ask the member of could team?